cartfile

```diff
@@ -0,0 +1,2 @@
+github "Quick/Quick" == 0.3.1
```

Comment:
Can you capitalize the name of this file, so it's `Cartfile` instead of `cartfile`?